### PR TITLE
fix(yamux): Schedule immediate wake-up instead of polling

### PR DESCRIPTION
**Pull request by thomaseizinger**
## Change checklist

- [ ] I have performed a self-review of my own code
- [ ] I have made corresponding changes to the documentation
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] A changelog entry has been made in the appropriate crates

### CODE REVIEW

File: muxers/yamux/CHANGELOG.md [hunk 1]
```diff
@@ -1,3 +1,12 @@
+# 0.41.1 [unreleased]
```
Comment (mxinden) [inline muxers/yamux/CHANGELOG.md]:
```suggestion
# 0.41.1
```

I will cut a release right away.

**Review by mxinden — APPROVED** [review]
Thanks @thomaseizinger!

**Comment by mxinden** [comment]
[Tagged](https://github.com/libp2p/rust-libp2p/releases/tag/libp2p-yamux-v0.41.1) and [published](https://crates.io/crates/libp2p-yamux/0.41.1).

//CC @rkuhn

**Comment by rkuhn** [comment]
Thanks!

**Pull request merged**